test/js/bun/console/console-table.test.ts

```diff
@@ -157,3 +157,24 @@ test("console.table json fixture", () => {
   expect(actualOutput).toMatchSnapshot();
   console.log(actualOutput);
 });
+
+test.skip("console.table character widths", () => {
+  // note: this test cannot be automated because cannot test printed witdhs consistently.
```

Comment:
Not sure I understand this, why can't we take a snapshot of the stdout?

Reply:
Also, comment should be "widths"

Reply:
The unicode characters that are printed will be always output the same bytes, so a snapshot would not help.
What the test should look at is how they are actually visually printed in a terminal/tty, but there's no easy way to do that, even more f it has to support running on CI without actual visual terminals

Reply:
Even so, anyways how it actually ends up printed is gonna depend on the font configured by the user, the fallback fonts in the OS, etc.
So the safest bet is to follow the Unicode specs on width